test/core/sector_storage/scheduler_test.cpp

```diff
@@ -246,12 +246,12 @@ namespace fc::sector_storage {
 
     std::mutex res;
     std::condition_variable cv;
-    bool ready = false;
+    std::atomic_bool ready = false;
 
     CallId call_id1{.sector = sector_id1, .id = "UUID1"};
     WorkerAction work1 = [&](const std::shared_ptr<Worker> &worker) {
       std::unique_lock<std::mutex> l(res);
-      cv.wait(l, [&]() { return ready; });
```

Comment:
maybe change "condition_variable" to "promise" from "future"

Reply:
done